src/nanocurrency/work.py

```diff
@@ -27,16 +27,19 @@
 #       time?
 _cpu_flags = cpuinfo.get_cpu_info()["flags"]
 
-if "avx" in _cpu_flags:
-    from . import _work_avx as _work
-elif "sse4_1" in _cpu_flags:
-    from . import _work_sse4_1 as _work
-elif "ssse3" in _cpu_flags:  # SSSE3
-    from . import _work_ssse3 as _work
-elif "sse2" in _cpu_flags:  # SSE2
-    from . import _work_sse2 as _work
-else:
-    from . import _work_ref as _work
+try:
+    if "avx" in _cpu_flags:
+        from . import _work_avx as _work
+    elif "sse4_1" in _cpu_flags:
+        from . import _work_sse4_1 as _work
+    elif "ssse3" in _cpu_flags:  # SSSE3
+        from . import _work_ssse3 as _work
+    elif "sse2" in _cpu_flags:  # SSE2
+        from . import _work_sse2 as _work
+    else:
+        from . import _work_ref as _work
+except:
```

Comment:
Silently ignoring errors in this section means the library will fail if someone tries to generate PoW later on, with little information on *why* it is failing. In this case it's likely that the user hasn't installed the library, in which case failing here is the correct thing to do.

Don't use bare except. Catch `ImportError` here if you need to watch out for imports failing, but that isn't what we need to do here.